.gitignore

```diff
@@ -15,3 +15,4 @@ StyleCop.Cache
 **/lcov.info
 coverage.json
 *.opencover.xml
+/listbackup/LISTS
```

Comment:
Just curious: where is this coming from?

It may be worth leaving a #comment here, but not a big deal

Reply:
My bad, I had put a folder in the root folder of the local repo and didn't want that to be included in the remote so I added it to the git ignore.

Reply:
Configure yourself a global gitignore
https://docs.github.com/en/github/using-git/ignoring-files

Don't commit stuff that only affects you to the repo please.